wdyr.js

```diff
@@ -0,0 +1,23 @@
+// Implements Why Did You Render (WDYR) in Dev
+
+import React from 'react';
+import Config from 'react-native-config';
+import lodashGet from 'lodash/get';
+
+const useWDYR = lodashGet(Config, 'USE_WDYR') === 'true';
+
+if (useWDYR) {
+    const whyDidYouRender = require('@welldone-software/why-did-you-render');
+    whyDidYouRender(React, {
+        // Tracks all components by default
+        trackAllPureComponents: true,
```

Comment:
Should this be `false` (the default)? PureComponent seems like the least likely to be re-rendering for bad reasons?

Reply:
After doing more research and seeing [this comment from one of the creators](https://github.com/welldone-software/why-did-you-render/issues/182), it seems that tracking pure components by default is a better idea, as they might control big render trees.

So I will keep that as the default, but I also included a way to enable tracking in all components, as well as examples to enable tracking by component `displayName`.

`Screen` must be excluded when tracking all components to avoid the following error that crashes the app:

`Uncaught Error: A navigator can only contain 'Screen', 'Group' or 'React.Fragment' as its direct children (found 'WDYRFunctionalComponent' for the screen 'Home'). To render this component in the navigator, pass it in the 'component' prop to 'Screen'.`

Reply:
Ok that works. Weird about the `Screen` component. Must have used this last before we added react navigation.